openeo-geotrellis/src/test/scala/org/openeo/geotrellis/MergeCubesSpec.scala

```diff
@@ -121,6 +144,63 @@ class MergeCubesSpec {
     assertTrue(mse < 0.1)
   }
 
+
+  @ParameterizedTest
+  @MethodSource(Array("testMergeCubesTiledNodataArguments"))
+  def testMergeCubesTiledNodata(aggregateR: AggregationType.Value,
+                                aggregateG: AggregationType.Value,
+                                aggregateB: AggregationType.Value,
+                               ): Unit = {
+    val path = "tmp/testMergeCubesTiledNodata/" + aggregateR + aggregateG + aggregateB + "/"
```

Comment:
JUnit 5 has `@TempDir` for this.

Reply:
I like to use the ./tmp folder, as it is easaly findable in the Jenkins workspace, and it it gitignored